src/ext/popup-menu.lisp

```diff
@@ -1,4 +1,4 @@
-(defpackage :lem/popup-menu
+q(defpackage :lem/popup-menu
```

Comment:
"q"

Reply:
Weird that it compiled with both of these typos. Fixed now.